lib/gui/api/index.ts

```diff
@@ -1,22 +1,19 @@
 import {ApiFacade} from './facade';
-import type Testplane from 'testplane';
 import {Express} from 'express';
 
 export interface ServerReadyData {
     url: string;
 }
 
-type TestplaneWithGui = Testplane & { gui: ApiFacade };
-
-export class Api {
+export class GuiApi {
     private _gui: ApiFacade;
 
-    static create<T extends Api>(this: new (testplane: TestplaneWithGui) => T, testplane: TestplaneWithGui): T {
-        return new this(testplane);
+    static create<T extends GuiApi>(this: new () => T): T {
+        return new this();
     }
 
-    constructor(testplane: TestplaneWithGui) {
-        this._gui = testplane.gui = ApiFacade.create();
```

Comment:
Tool is not passed here anymore. Because I don't need to set `gui` for other tools (pwt for example). Now it should be specified inside tool adapter instance.